### ISSUE TITLE
yazl options

### ISSUE THREAD

**Issue by mabar** [joaomoreno/gulp-vinyl-zip#7]
Please add a way to pass yazl options. I'm currently using gulp-vinyl-zip for backup of my personal projects (because I don't need CVS) and compression is only slowing me down.

**Comment by joaomoreno** [comment]
Want to submit a PR?

**Comment by mabar** [comment]
Sorry, I am new in world of JS. But it looks like this can help:

```
function zip(zipPath, compress = true) {
   //some code
   var opts = {
      mtime: stat.mtime,
      mode: stat.mode,
      compress: compress
    };
```

```
function dest(zipPath, opts, compress = true) {
    var input = zip(path.basename(zipPath), compress);
```

No breaking change and compression can be turned off.

**Comment by joaomoreno** [comment]
Fixed in 1.4.0:

``` javascript
...
  .pipe(zip.dest('out.zip', { compress: false }));
```
